kythe/go/extractors/config/runextractor/backup/backup.go

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2018 The Kythe Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package backup is a simple library for backing up a config file and restoring
+// it using a temporary file.
+package backup
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+)
+
+// Backup copies a given file to a temp file, returning the temp file name.
+func Backup(configFile string) (tfname string, oerr error) {
```

Comment:
Since the package is already called `backup`, the name `Backup` seems redundant. 

Perhaps `backup.Save` to match `backup.Restore`?